File: Backend/DisasterDispatch.Core/Entities/CustomOperation.cs
using DisasterDispatch.Core.Entities.Common;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DisasterDispatch.Core.Entities
{
    public class CustomOperation:BaseEntity
    {
        public Guid EmergencyReportId{ get; set; }
        public EmergencyReport EmergencyReport{ get; set; }
        public Guid DisasterOperationId{ get; set; }
        public DisasterOperation DisasterOperation{ get; set; }
        public ICollection<AppUser> Workers{ get; set; }
        public string Status{ get; set; }

    }
}

File: Frontend/DisasterDispatchFrontend/src/app/Models/custom-operation.ts
import { Guid } from "guid-typescript";
import { EmergencyReport } from "./emergency-report";
import { DisasterOperation } from "./disaster-operation";
import { AppUser } from "./app-user";
import { BaseEntity } from "./base-entity";

export class CustomOperation extends BaseEntity {
    emergencyReportId:Guid;
    emergencyReport:EmergencyReport;
    disasterOperationId:Guid;
    disasterOperation:DisasterOperation;
    workers:AppUser[];
    status:string;
}
